roles/lib_openshift/src/ansible/oc_adm_registry.py

```diff
@@ -0,0 +1,47 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module for registry
+    '''
+
+    module = AnsibleModule(
```

Comment:
@ashcrow We already have some major issues that prevent backporting changes to 3.1, I see no reason why we would need to maintain <= 3.1 support for this.